install/kubernetes/addons/grafana.yaml

```diff
@@ -1,52 +0,0 @@
----
-apiVersion: v1
```

Comment:
why remove these?

Reply:
updateVersion is creating them from templates - but with slightly different content. 
Someone already removed the other generated yaml files, but forgot those 2.

Reply:
The deletion of this is already queued to the merge bot via https://github.com/istio/istio/pull/3183